dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -2075,6 +2100,54 @@ def  put_participant_in_section(student, teacher, script)
     assert extras_data['bonusLevels'][0]['levels'][0]['perfect']
   end
 
+  test "lesson extras shows instructor progress for participant if section and user id" do
```

Comment:
does this mean "lesson extras shows progress to instructor for ..."? if so I'd recommend that wording, because this wording makes me think "...shows progress of instructor..."

Reply:
Good call - Updated!